src/components/Article.jsx

```diff
@@ -28,15 +28,38 @@ export default function Article() {
             setComments(data.comments)
         })
         .catch(err => console.log("error", err))
-    }, [article_id]) 
+    }, [article_id, comments]) 
+
+
+    let [commentInput, setCommentInput] = useState("")
+    let [commentError, setCommentError] = useState(false)
 
     if (loadingArticle) return <p>loading</p>
 
+    function submitComment(event) {
+        event.preventDefault()
+        axios.post(`https://nc-news-backend-shahbazrafi.herokuapp.com/api/articles/${article_id}/comments`, {username: username, body: commentInput})
```

Comment:
Please extract your api calls out to one file. This component is getting very long and difficult to follow.